lib/internal/modules/esm/loader.js

```diff
@@ -408,41 +474,96 @@ class ESMLoader {
 
     const namespaces = await PromiseAll(new SafeArrayIterator(jobs));
 
-    return wasArr ?
-      namespaces :
-      namespaces[0];
+    if (!wasArr) return namespaces[0];
```

Comment:
```suggestion
    if (!wasArr) {
      return namespaces[0];
    }
```

Reply:
I'm happy to do, but can I do these in a tidy PR?

Reply:
These are pretty trivial tweaks, shouldn’t they happen in this PR? I can submit a PR against your PR if you’d like.

Reply:
It's not about the effort (which is indeed small); I'm thinking for consistency to get them all at once.

Reply:
It’s my opinion that we should have a linting rule preventing `if` (or `for`, `while` etc.) blocks without braces. It’s this one: https://eslint.org/docs/rules/curly. I’ve seen bugs come from failing to follow this rule.

Presumably we don’t have this enabled for the entire codebase, and enabling such a rule definitely needs to be its own PR because there are probably cases scattered throughout the codebase that would need updating as a result. So for the time being I’m just trying to prevent _new_ instances of violations of this rule, until it can be enabled in our ruleset. cc @aduh95 @Trott (Also, please let me know if this has already been discussed and the consensus is otherwise.)

As far as this PR goes, I’m just trying to follow this rule for all the new lines added, ignoring existing code. It would be easier to backport chaining if it lands as a single commit rather than a commit plus a cleanup commit.

Reply:
Sure, I'm fine with applying or enabling that rule as described. Right now, I think the 1-liners are most prevalent. I was envisioning something similar for these. The clean-up would be more than just things related to this PR (which is why I wanted to do it then); ex clean up autofix mangling, add code docs.

If you think it's better to do them now, sure.